contracts/exchange/contracts/src/interfaces/IExchangeRichErrors.sol

```diff
@@ -1,30 +1,12 @@
-/*
-
-  Copyright 2018 ZeroEx Intl.
-
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
-
-*/
-
 pragma solidity ^0.5.9;
 
-import "@0x/contracts-utils/contracts/src/mixins/MRichErrorTypes.sol";
-import "@0x/contracts-exchange-libs/contracts/src/LibOrder.sol";
 
+contract IExchangeRichErrors {
+    enum AssetProxyDispatchErrorCodes {
```

Comment:
Styling nit: Add a newline after `contract IExchangeRichErrors {` (I've seen this in a few other places as well).

Reply:
Do we need to put a new line after every contract declaration, or only when the declaration fits on a singe line?

Ex. Would the following need a new line after `{`?

```
contract A is 
    B,
    C
{
    enum D { }
}

Reply:
Only after the declaration fits on a single line (so you wouldn't need one in your example).